kyuubi-zookeeper/src/main/scala/org/apache/kyuubi/zookeeper/ZookeeperConf.scala

```diff
@@ -56,12 +56,14 @@ object ZookeeperConf {
     .version("1.2.0")
     .fallbackConf(EMBEDDED_ZK_TEMP_DIR)
 
+  @deprecated(s"the config will be removed", since = "1.3.0")
```

Comment:
I guess all these configs can be removed since they don't print any warning or blocks anything after being removed

Reply:
Yeah, they don't print any warning or blocks anything after being removed. We keep these as @pan3793 suggested.

Reply:
If we choose to keep this, we need to notify users at least; otherwise, they are completely dead code

Reply:
cc @pan3793

Reply:
OK, let's remove it.

Reply:
Done.